tetrateci/test_1.7.sh

```diff
@@ -13,17 +14,14 @@ go test -count=1 -tags=integ ./tests/integration/pilot/analysis/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/pilot/revisions/... -p 1 -test.v
 go test -count=1 -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics' ./tests/integration/telemetry/stats/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/ca_custom_root/... -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/chiron/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/filebased_tls_origination/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/mtls_first_party_jwt/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/mtlsk8sca/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/sds_egress/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/sds_tls_origination/... -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/webhook/... -p 1 -test.v
 
-if [[ ${CLUSTER} == "eks" ]]; then
-  go test -count=1 -tags=integ ./tests/integration/security/chiron/... -p 1 -test.v
-fi
-
```

Comment:
This file should look like the one in #25, but I could address it after merge #24 since I will need to port those changes to this branch.